app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivityTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.github.h3lp3rs.h3lp.forum
+
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.replaceText
+import androidx.test.espresso.intent.Intents.init
+import androidx.test.espresso.intent.Intents.release
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.USER_TEST_ID
+import com.github.h3lp3rs.h3lp.forum.data.ForumPostData
+import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import junit.framework.Assert.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.kotlin.any
+import java.time.ZonedDateTime
+
+const val CATEGORY_TEST_STRING = "GENERAL"
+val CATEGORY_TEST = ForumCategory.GENERAL
+const val QUESTION_TEST = "question"
```

Comment:
Limit the scope to private in order to not pollute the environment (add as companion object in class for ex)

Reply:
They are not private because they are used in other classes

Reply:
You can put it as test package visibility then ;)